core-lib/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Add core-lib
+zephyr_include_directories(include)
```

Comment:
This PR should be the first because of dependencies.

Reply:
updated